; Plan generado automaticamente - 2025-08-18 14:37:54.792661
; Dominio: cement-alimentacion.pddl
; Problema: cement_problem.pddl

0.00000: (alimentar-clinker mc3 t3-clinker mc3-desde_silo-blanco)
4.01000: (alimentar-puzolana-s mc3 t3-puzolana-s ps-a-mc3-por-mc2)
Planning service: https://solver.planning.domains:5001/package/delfi/solve
Domain: cement-alimentacion, Problem: cement-production-problem
Parsing pddl.....
==== Computing an abstract structure graph from the lifted task description... ====
Graph computation returned nonzero exitcode 1
==== Computing abstract structure graph failed, using fallback planner! ====
==== Running fallback planner... ====
Running planner, call string: ['/usr/bin/python', '/workspace/delfi/fast-downward.py', '--transform-task', 'preprocess', '--build', 'release64', '--search-memory-limit', '7600M', '--plan-file', 'plan', 'domain', 'problem', '--symmetries', 'sym=structural_symmetries(search_symmetries=dks)', '--search', 'astar(celmcut,symmetries=sym,pruning=stubborn_sets_simple(minimum_pruning_ratio=0.01),num_por_probes=1000)']
INFO     Running translator.
INFO     translator input: ['domain', 'problem']
INFO     translator arguments: []
INFO     translator time limit: None
INFO     translator memory limit: None
INFO     callstring: /usr/bin/python /workspace/delfi/builds/release64/bin/translate/translate.py domain problem
Parsing...
Command '['/usr/bin/python', '/workspace/delfi/builds/release64/bin/translate/translate.py', 'domain', 'problem']' returned non-zero exit status 1
Overall time: [0.050s CPU, 0.059s wall-clock]
==== Done running fallback planner. ====
Error: Using TensorFlow backend.
Traceback (most recent call last):
File "/workspace/delfi/builds/release64/bin/translate/abstract_structure_module.py", line 547, in <module>
task = pddl_parser.open()
File "/workspace/delfi/builds/release64/bin/translate/pddl_parser/pddl_file.py", line 37, in open
return parsing_functions.parse_task(domain_pddl, task_pddl)
File "/workspace/delfi/builds/release64/bin/translate/pddl_parser/parsing_functions.py", line 308, in parse_task
= parse_domain_pddl(domain_pddl)
File "/workspace/delfi/builds/release64/bin/translate/pddl_parser/parsing_functions.py", line 360, in parse_domain_pddl
requirements = pddl.Requirements(opt[1:])
File "/workspace/delfi/builds/release64/bin/translate/pddl/tasks.py", line 70, in __init__
":derived-predicates", ":action-costs"), req
AssertionError: :durative-actions
Traceback (most recent call last):
File "/workspace/delfi/builds/release64/bin/translate/translate.py", line 753, in <module>
main()
File "/workspace/delfi/builds/release64/bin/translate/translate.py", line 730, in main
domain_filename=options.domain, task_filename=options.task)
File "/workspace/delfi/builds/release64/bin/translate/pddl_parser/pddl_file.py", line 37, in open
return parsing_functions.parse_task(domain_pddl, task_pddl)
File "/workspace/delfi/builds/release64/bin/translate/pddl_parser/parsing_functions.py", line 308, in parse_task
= parse_domain_pddl(domain_pddl)
File "/workspace/delfi/builds/release64/bin/translate/pddl_parser/parsing_functions.py", line 360, in parse_domain_pddl
requirements = pddl.Requirements(opt[1:])
File "/workspace/delfi/builds/release64/bin/translate/pddl/tasks.py", line 70, in __init__
":derived-predicates", ":action-costs"), req
AssertionError: :durative-actions
Planner output: {}No plan found in the planner output.